src/main/resources/openapi/specs/api.kuflow.com/v1/openapi.yaml

```diff
@@ -268,19 +268,31 @@ paths:
                     "activityResponseVersion": "string",
                     "elementValues":
                       {
-                        "element_code_1": { "value": "string" },
-                        "element_code_2": { "valid": false, "value": 1111 },
-                        "element_code_3": [{ "value": "string" }, { "value": "string" }],
-                        "element_code_4": { "value": { "form_field_1": "string", "form_field_2": "string" } },
-                        "element_code_5": { "value": { "id": "string", name: "", contentPath: "", contentType: "", contentLength: "" } },
-                        "element_code_6":
-                          [{ "value": { "value": { "id": "string", name: "", contentPath: "", contentType: "", contentLength: "" } } }],
-                        "element_code_7": { "value": { "id": "b90c9ae9-444c-4bfd-9594-7468955d2eda", "type": "USER" } },
-                        "element_code_8":
-                          [
-                            { "value": { "id": "b90c9ae9-444c-4bfd-9594-7468955d2eda", "type": "USER" } },
-                            { "value": { "id": "f4be313c-c0a2-426c-bdd3-c3113d7bbf37", "type": "USER" } },
-                          ],
+                        "element_code_1": { type: "STRING", "value": "string" },
+                        "element_code_2": { type: "NUMBER", "valid": false, "value": 1111 },
+                        "element_code_3": [
+                          { type: "STRING", "value": "string" },
+                          { type: "STRING", "value": "string" }
+                        ],
+                        "element_code_4": {
+                          type: "OBJECT",
+                          "value": { "form_field_1": "string", "form_field_2": "string" }
+                        },
+                        "element_code_5": {
+                          type: "DOCUMENT",
+                          "value": { "id": "string", name: "", contentPath: "", contentType: "", contentLength: 100 }
+                        },
+                        "element_code_6": [
+                          {
+                            type: "DOCUMENT",
+                            "value": { "id": "string", name: "", contentPath: "", contentType: "", contentLength: 100 }
+                          }
+                        ],
```

Comment:
in the example, maybe we can take advantage of and remove the contentType="" and other empty fields and leave only the id, what do you think?

Reply:
no problem, I'm going to remove it